### PR TITLE
Explain private keys and account funding

### PR DESCRIPTION
**Pull request by miohtama**
Some basics about private keys and using ETH as a gas fee. This information is unknown to many new developers.

### What was wrong?

People do not know how to use private keys in development.

### How was it fixed?

Explain the basics in the documentation.

#### Cute Animal Picture

```
((...))         ((...))         ((...))           L...L  
( o o )         ( x x )         ( O O )          < o o >
 \   /           \   /           \   /            \   /
  ^_^  cp97       ^_^  cp97      (`_`)  cp97       ^_^  cp97
```

### CODE REVIEW

File: docs/web3.eth.account.rst [hunk 1]
```diff
@@ -55,8 +55,55 @@ Using private keys usually involves ``w3.eth.account`` in one way or another. Re
...
+  to store the key. Read more below.
+  
+- You can also import the raw hex private key to MetaMask and any other
+  wallet - the private key can be shared between your Python code
+  and any number of wallets.
+
+Funding a New Account
+---------------------
+
+If you create a private key it comes with its own Ethereum address.
```
Comment (fselmo) [inline docs/web3.eth.account.rst:92]:
```suggestion
If you create a private key, it comes with its own Ethereum address.
```

**Review by fselmo — COMMENTED** [review]
Thanks for the much-needed improvement / clarification. Just had some minor typos / suggestions. I can make those if you're good with it and add a `doc` newsfragment... lmk 👀

**Comment by miohtama** [comment]
@fselmo resolved spell check - thank you!

**Comment by fselmo** [comment]
rebased and consolidated commits 👍🏼

**Pull request merged**